bootstrap/developer_guide.md

```diff
@@ -31,5 +31,38 @@ go install github.com/kubeflow/kubeflow/bootstrap/cmd/bootstrap
 or
 
 ```sh
-go build -i -o /opt/kubeflow/bootstrapper ${GOPATH}/src/github.com/kubeflow/kubeflow/bootstrap/cmd/bootstrap/main.go
-```
\ No newline at end of file
+go build -i -o ${GOPATH}/bin/bootstrapper ${GOPATH}/src/github.com/kubeflow/kubeflow/bootstrap/cmd/bootstrap/main.go
+```
+
+## Building and pushing the bootstrapper-builder and bootstrapper images.  
+
+The bootstrapper image defaults to [gcr.io/kubeflow-images-public/bootstrapper](https://gcr.io/kubeflow-images-public/bootstrapper). 
+This image inherits from a base "builder" image which defaults to 
+[gcr.io/kubeflow-images-public/bootstrapper-builder](https://gcr.io/kubeflow-images-public/bootstrapper-builder). 
```

Comment:
Is this true? I thought the builder was just used to build the binaries for the bootstrapper?